<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sendmessagegroup_Dialog</class>
 <widget class="QDialog" name="sendmessagegroup_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1254</width>
    <height>723</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0.019, y1:0.017, x2:1, y2:1, stop:0 rgba(85, 65, 4, 255), stop:0.16 rgba(119, 93, 19, 255), stop:0.225 rgba(148, 125, 37, 255), stop:0.285 rgba(180, 160, 65, 255), stop:0.342723 rgba(195, 183, 96, 255), stop:0.408028 rgba(218, 212, 118, 255), stop:0.52 rgba(224, 205, 93, 255), stop:0.57 rgba(199, 164, 46, 255), stop:0.635 rgba(168, 142, 42, 255), stop:0.695 rgba(180, 155, 61, 255), stop:0.75 rgba(207, 191, 82, 255), stop:0.815 rgba(208, 187, 73, 255), stop:0.88 rgba(187, 156, 51, 255), stop:0.935 rgba(76, 60, 14, 255), stop:1 rgba(35, 40, 3, 255));
color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QGroupBox" name="groupname_groupBox">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>240</y>
     <width>461</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>groupname :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="groupname_lineEdit">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="back_pushButton">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>back</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="confirm_pushButton">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>confirm</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="chat_groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>1231</width>
     <height>701</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLineEdit" name="send_lineEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>name :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="name_label">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="send_pushButton">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>send</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTextEdit" name="message_textEdit">
      <property name="styleSheet">
       <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 255, 0, 69), stop:0.375 rgba(255, 255, 0, 69), stop:0.423533 rgba(251, 255, 0, 145), stop:0.45 rgba(247, 255, 0, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.55 rgba(255, 255, 0, 255), stop:0.57754 rgba(255, 203, 0, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69));
font: 11pt &quot;Segoe UI&quot;;
color: rgb(97, 49, 0);</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="back_chat_pushButton">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>back</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>chat_groupBox</zorder>
  <zorder>groupname_groupBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
